metaflow/cli.py

```diff
@@ -90,15 +90,21 @@ def echo_always(line, **kwargs):
         click.secho(ERASE_TO_EOL, **kwargs)
 
 
-def logger(body='', system_msg=False, head='', bad=False, timestamp=True):
+def logger(body='',
+           system_msg=False,
+           head='',
+           bad=False,
+           timestamp=True,
+           err=False):
```

Comment:
I am leaning towards setting the default value of `err` to True. Thoughts, @tuulos ?